libr/core/bin.c

```diff
@@ -268,6 +268,9 @@ static int bin_info (RCore *r, int mode) {
 	char str[R_FLAG_NAME_SIZE];
 	RBinInfo *info = r_bin_get_info (r->bin);
 	RBinFile *binfile = r_core_bin_cur (r);
+	char size_str[11];
+	snprintf (size_str, 11, "%d",  r_bin_get_size (r->bin));
```

Comment:
use sizeof(size_str), this way it will be correct, even if you change the size of size_str.
Also, you can put that definition of size_str at the begining of the function